.github/workflows/build_BE.yml

```diff
@@ -87,19 +87,29 @@ jobs:
           fail_ci_if_error: false
           verbose: true
 
-      - name: Creating Lint + Bug Reports
+      - name: Creating Lint + Bug Reports - Back-End
         run: |
           rm -f ../*_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../lint_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../bug_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../BE_lint_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../BE_bug_report.txt
         shell: bash
-
+     
+      - name: Creating Lint + Bug Reports - Front-End
+        working-directory: ./frontend
+        run: |
+          rm -f ../FE_bug_report.html
+          rm -f ../FE_lint_report.json
+          npx eslint . -f node_modules/eslint-html-reporter/reporter.js -o ../FE_bug_report.html
+          npx standard | npx standard-reporter --json > ../FE_lint_report.json
```

Comment:
Strange...for some reason eslint isn't working for me in development.

Reply:
yea not workign for me either. Going to redo the installation myself so i know the context